apps/src/templates/button.module.scss

```diff
@@ -0,0 +1,238 @@
+@import "color.scss";
+
+$button-height-default: 34px;
+$button-height-large: 40px;
+$button-height-narrow: 40px;
+$button-height-inline: 20px;
+
+.main {
```

Comment:
we have some old styles on the backend that are loaded on every page (these will eventually go away) that are overriding some of these styles now that we're using classes instead of just clobbering all styles with the `style` HTML attr. the button styles live here:

https://github.com/code-dot-org/code-dot-org/blob/3f3f1b3e70a15efdefb75553e58689734164df54/dashboard/app/assets/stylesheets/application.scss#L3058

so in order for our styles to take precedence, they need to be more specific. we can do this by wrapping the styles here in tags, which is unfortunate, but not the worst and won't create cascading issues:

```scss
div,
button,
a,
a:link,
a:visited {
  // classes nested in here
}
```

the caveat being that not all styles should be in that block because we have some styles (like `moduleStyles.icon`) that would get lost if we did that

Reply:
It took me a while, but I think I finally figured out how to have this apply correctly. The teacher dashboard http://localhost-studio.code.org:3000/home has some good edge cases that were helpful to test with.